app/components/layout/sidebar/atoms.ts

```diff
@@ -1,8 +1,8 @@
 import { atom } from "jotai";
 
-export const sidebarCollapseStatusAtom = atom(false);
```

Comment:
I had changed those as I prefer the default state to be open.

Reply:
Okay wasn't aware of that. Will use different atoms for mobile.